bitnami/kafka/values.yaml

```diff
@@ -802,3 +802,16 @@ externalZookeeper:
   ## Server or list of external zookeeper servers to use.
   ##
   servers: []
+
+## Kafka Log4J Configuration
+## The log4j.properties defaults to the values in the Kafka release when this paremeter is not specified.
+## See an example log4j.properties at:
+## https://github.com/apache/kafka/blob/trunk/config/log4j.properties
+##
+# log4j:
+
+## Kafka Log4j ConfigMap
+## The name of an existing ConfigMap containing a log4j.properties file.
+## NOTE: this will override log4j.
+##
+# log4jConfigMap:
```

Comment:
Please use `existingLog4jConfigMap` for consistency in the naming with the other existing parameter `existingConfigMap`